superset/assets/javascripts/explorev2/reducers/exploreReducer.js

```diff
@@ -90,6 +87,7 @@ export const exploreReducer = function (state, action) {
           chartUpdateEndTime: null,
           chartUpdateStartTime: now(),
           triggerQuery: false,
+          queryRequest: action.queryRequest,
```

Comment:
I personally like putting the request into state, I don't think it's too much anti-pattern, since it's just a request object, on the other hand, we have mapStateToFields methods attached with all our fields in state as well.